### DISCUSSION TITLE
Error while running php artisan key:generate command on ubuntu

### DISCUSSION THREAD

**Discussion by arjunks94** [snipe/snipe-it#11749] [Q&A] [answered]
In Request.php line 334:
                                                                               
  Symfony\Component\HttpFoundation\Request::create(): Argument #1 ($uri) must  
   be of type string, null given, called in /home/dohsiit/Snipe-IT-DOHSI/snip  
  e-it/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/SetReques  
  tForConsole.php on line 32                                                   
                                
![Screenshot from 2022-08-27 18-57-23](https://user-images.githubusercontent.com/84993460/187038027-88ce681a-8711-422c-be38-cad2d6b030b2.png)

**Comment by miklosania** [comment, marked as answer]
Same here on Windows IIS:
![image](https://user-images.githubusercontent.com/89782874/187178294-f42ac4bd-ad32-47a8-bd3d-1677227636c5.png)

Can anyone please help with this?

**Reply by arjunks94** [reply]
@snipe please assist for a solution

**Reply by miklosania** [reply]
https://github.com/snipe/snipe-it/issues/11498#issuecomment-1181699763
It solved the problem for me!

**Reply by arjunks94** [reply]
That issue is resolved now the preflight screen is coming blank i have reverified all configuration nothing is wrong
